lib/renderer/init.js

```diff
@@ -121,7 +121,7 @@ if (preloadScript) {
   try {
     require(preloadScript)
   } catch (error) {
-    if (error.code === 'MODULE_NOT_FOUND') {
+    if (error.code === 'MODULE_NOT_FOUND' && error.message.indexOf(preloadScript) > -1) {
```

Comment:
Good point, hmm, I wonder if in all cases then it should just log the same thing, the stack might be helpful for both cases.

So what do you think about something like:

```js
} catch (error) {
  console.error('Unable to load preload script: ' + preloadScript)
  console.error(error.stack || error.message)
}
```